XenAdmin/ConsoleView/XSVNCScreen.cs

```diff
@@ -1386,5 +1385,31 @@ private bool Keysym(bool pressed, Keys key, Keys extendedKey)
 
             return false;
         }
+
+        private void InitializeComponent()

```

Comment:
I think this was added while experimenting with events on this control, but is not needed anymore.